AngularMomentum/OpenSim_get_angular_momentum_from_body_kin_new.m

```diff
@@ -0,0 +1,120 @@
+function [whole_body_L,data_out,headers,time,bodymass] = OpenSim_get_angular_momentum_from_body_kin_new(model_path,BodyKin)
+% OPENSIM_GET_ANGULAR_MOMENTUM Computes the angular momentum for each
+% segment based on a OpenSim model and his kinematics (based on paper Herr
+% et al. 2008, Control of angular momentum during walking)
+%
+%   Input Arguments:
+%       -1. model_path => path to the musculoskeletal model
+%       -2. ik_filepath => path to the file with the kinematics
```

Comment:
Inputs arguments are not correct (you have model_path and bodykin as input to these functions)